app/code/Transoft/Blog/Api/ModelRepositoryInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Api;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Transoft\Blog\Api\Data\ModelInterface;
+use Transoft\Blog\Api\Data\ModelSearchResultsInterface;
+
+interface ModelRepositoryInterface
+{
+    /**
+     * @param ModelInterface $model
+     * @return ModelInterface
+     * @api
```

Comment:
Please remove @api annotation. First of all, it is added to a class, not to method. Second - you did not define any API.

Reply:
removed